app/javascript/packages/transliterable-field-group/index.ts

```diff
@@ -0,0 +1,375 @@
+import { trackError } from '@18f/identity-analytics';
+import { request } from '@18f/identity-request';
+
+/**
+ * Representation of the response from a transliteration API call
+ */
+interface ValidationResponse {
+  success: boolean;
+  data: Record<string, string>;
+}
+
+/**
+ * Internal cache structure for controlling when API call is made
+ * and reusing result
```

Comment:
what is meant by reusing result here? is it when the cache is used?

Reply:
The results from unchanged fields are retained internally and reset on the inputs if needed (as opposed to querying the API redundantly).

Reply:
ahh ok!